gcc/rust/lang.opt

```diff
@@ -113,6 +113,48 @@ Rust Joined RejectNegative
 
 o
 Rust Joined Separate
+
+frust-compile-until=
+Rust Joined RejectNegative Enum(frust_compile_until) Var(flag_rust_compile_until)
+-frust-compile-until=[2015|2018|2021]             When to stop in the pipeline when compiling Rust code
```

Comment:
looks like the help string is wrong here

Reply:
Ah, thank you :) Fixed